modules/gitops-additional-steps-for-disconnected-clusters.adoc

```diff
@@ -0,0 +1,45 @@
+[id="gitops-additional-steps-disconnected-clusters_{context}"]
+= Integrating Keycloak with the OpenShift OAuth server in a disconnected cluster
+
+In a disconnected cluster, Keycloak communicates with OpenShift OAuth server through a proxy. 
```

Comment:
the OpenShift OAuth server - please add the article 'the' in all places you reference OpenShift OAuth server